### DISCUSSION TITLE
[Feature Request] Glances widget for CPU, Ram, storage on homelab

### DISCUSSION THREAD

**Discussion by mgrimace** [glanceapp/glance#283] [General]
This is a request for Glances (plural) widgets, from: https://github.com/nicolargo/glances.
Specifically, I use CPU, Ram, and NAS storage monitoring for my small homelab in the form of widgets, like: https://gethomepage.dev/widgets/services/glances/#views

It’s handy to see free storage for example at a glance on my dash page. Glance (singular) has become my default dashboard/homepage, and this would add some amazingly useful info about my system at-a-glance. 

Thanks for all your work!

**Comment by nikolasdmtr** [comment]
In lack of another option, you can do this: https://github.com/glanceapp/glance/discussions/192
